test/AsmResolver.DotNet.Tests/Collections/ParameterCollectionTest.cs

```diff
@@ -111,6 +118,18 @@ public void ReadMultipleParametersFromInstanceMethod()
             Assert.Equal(nameof(InstanceMethods), method.Parameters.ThisParameter.ParameterType.Name);
         }
 
+        [Fact]
+        public void ReadEmptyParametersFromGenericInstanceMethod()
+        {
+            var method = ObtainGenericInstanceTestMethod(nameof(GenericInstanceMethods<int, int>.InstanceParameterlessMethod));
+            Assert.Empty(method.Parameters);
+            Assert.NotNull(method.Parameters.ThisParameter);
+            var genericInstanceType = method.Parameters.ThisParameter.ParameterType as GenericInstanceTypeSignature;
+            Assert.NotNull(genericInstanceType);
+            Assert.True(genericInstanceType.TypeArguments.Count == 2);
+            Assert.Equal("GenericInstanceMethods`2", genericInstanceType.GenericType.Name);
```

Comment:
Should we also test for the type arguments all being the right generic type parameter signatures pointing to the right indices?